src/components/timeline/TrackContextMenu.js

```diff
@@ -993,12 +1034,15 @@ class TimelineTrackContextMenuImpl extends PureComponent<
     const isolateProcess = this.renderIsolateProcess();
     const isolateLocalTrack = this.renderIsolateLocalTrack();
     const isolateScreenshot = this.renderIsolateScreenshot();
+    const showLocalTracksInProcess = this.renderShowLocalTracksInThisProcess();
     const hideTrack = this.renderHideTrack();
     const separator =
       isolateProcessMainThread ||
       isolateProcess ||
       isolateLocalTrack ||
-      isolateScreenshot ? (
+      isolateScreenshot ||
+      showLocalTracksInProcess ||
+      hideTrack ? (
```

Comment:
I've seen that `hideTrack` wasn't present here: you'll see this change in a test later too, but I couldn't understand in which case only `hideTrack` would be displayed -- it's possible because this happens in a test, but couldn't reproduce with a real profile.